File: android/app/src/main/java/com/hisab/app/domain/Currency.kt
package com.hisab.app.domain

/**
 * An explicit currency rather than an assumption baked into the code, even
 * though V1 only supports BDT — DECISIONS.md D019, PRD.md section 24
 * (Shop.currency).
 */
enum class Currency {
    BDT,
}

val DEFAULT_CURRENCY = Currency.BDT

File: server/src/domain/currency.ts
/**
 * An explicit currency field rather than an assumption baked into the code,
 * even though V1 only supports BDT — DECISIONS.md D019, PRD.md section 24
 * (Shop.currency).
 */
export type Currency = 'BDT'

export const DEFAULT_CURRENCY: Currency = 'BDT'
